### ISSUE TITLE
expected kubernetes_version lingering

### ISSUE THREAD

**Issue by linecolumn** [yamamoto-febc/terraform-provider-rke#95]
rke provider is lingering in regard latest rke supported k8s versions.

```
$ terraform plan

Error: expected kubernetes_version to be one of [v1.11.9-rancher1-1 v1.12.7-rancher1-2 v1.13.5-rancher1-2 v1.14.1-rancher1-1], got v1.13.5-rancher1-3

  on main.tf line 2, in resource "rke_cluster" "cluster":
   2: resource "rke_cluster" "cluster" {
```

rke reports "supported versions are: [v1.11.9-rancher1-2 v1.12.7-rancher1-3 v1.13.5-rancher1-3 v1.14.1-rancher1-2]"

**Comment by yamamoto-febc** [comment]
This provider is still linking to RKE v0.2.2.
We should upgrade to RKE v0.2.4.

**Comment by yamamoto-febc** [comment]
We released terraform-provider-rke v0.12.0.
https://github.com/yamamoto-febc/terraform-provider-rke/releases/tag/0.12.0

Please use it.
